modules/gapi/test/gapi_async_test.cpp

```diff
@@ -186,6 +253,12 @@ struct AsyncCompiled  : crtp_cast<crtp_final_t>{
         auto gcmpld = this->crtp_cast_(this)->compile();
         return cv::gapi::wip::async(gcmpld, std::forward<Args>(args)...);
     }
+
+    template<typename... Args>
+    auto async(cv::gapi::wip::GAsyncContext& ctx, Args&&... args) -> decltype(cv::gapi::wip::async(std::declval<cv::gapi::wip::GAsyncContext&>(), std::declval<cv::GCompiled&>(), std::forward<Args>(args)...)){
```

Comment:
Could you please divide this long expression on several lines? It's hard to review. This is also true for several similar instances in this test file below.

Reply:
split